functions/go/set-namespace/config/myapp.yaml

```diff
@@ -0,0 +1,36 @@
+#myapp/config/base/myapp.yaml
```

Comment:
Let's remove this file and use `kpt pkg get` for testing. Other functions get test configs using `kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-sdk/example-configs example-configs`.

Reply:
Done